<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="LAPTOP-1LD1TVSM" name="com.logintest.Test_001" tests="3" failures="0" timestamp="2022-07-13T12:08:02 IST" time="0.008" errors="1">
  <testcase name="verifyLoginpage" time="0.000" classname="com.logintest.Test_001">
    <skipped/>
  </testcase> <!-- verifyLoginpage -->
  <system-out/>
  <testcase name="StartSetup" time="0.008" classname="com.logintest.Test_001">
    <error type="java.io.FileNotFoundException" message="C:\Users\PRIYAL\eclipse-workspace\maven1\Testdata\data.xlsx (The system cannot find the file specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\PRIYAL\eclipse-workspace\maven1\Testdata\data.xlsx (The system cannot find the file specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(Unknown Source)
at java.io.FileInputStream.<init>(Unknown Source)
at java.io.FileInputStream.<init>(Unknown Source)
at com.utility.Excellentdataprovider.<init>(Excellentdataprovider.java:15)
at com.utility.Baseclass.StartSetup(Baseclass.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- StartSetup -->
  <system-out/>
  <testcase name="verifyLoginpage" time="0.000" classname="com.logintest.Test_001">
    <skipped/>
  </testcase> <!-- verifyLoginpage -->
  <system-out/>
</testsuite> <!-- com.logintest.Test_001 -->
